packages/ingest/src/HttpProviderClient.js

```diff
@@ -62,14 +64,35 @@ class HttpProviderClient {
 
     if (this.username) this.gotOptions.username = this.username;
     if (this.password) this.gotOptions.password = this.password;
+
+    const RedirectHandler = {
+      // Need to use named function and not fat arrow
+      // expression so that we can use the bound value
+      // of "this"
+      handleBeforeRedirect(options, response) {
+        if (redirectCodes.has(response.statusCode)) {
+          /* eslint-disable no-param-reassign */
+          options.url.username = this.username;
+          options.url.password = this.password;
+          /* eslint-enable no-param-reassign */
+        }
+      },
+    };
+    const boundHandleBeforeRedirect = RedirectHandler.handleBeforeRedirect.bind(this);
+    this.gotOptions.hooks = {
+      beforeRedirect: [
+        boundHandleBeforeRedirect,
+      ],
+    };
   }
 
   async downloadTLSCertificate() {
     if (!this.certificateUri || this.certificate !== undefined) return;
     try {
       const s3Params = parseS3Uri(this.certificateUri);
       this.certificate = await getTextObject(s3Params.Bucket, s3Params.Key);
-      this.gotOptions.ca = this.certificate;
+      this.gotOptions.https = this.gotOptions.https || {};
+      this.gotOptions.https.certificateAuthority = this.certificate;
```

Comment:
What was this change about?

Reply:
This was just to fix a warning from `got` because the `ca` option has been deprecated in favor of `https.certificateAuthority`